playground/src/docs.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { useEffect, useRef } from "preact/hooks";
+import { DocFile } from "../../npm/qsharp/dist/compiler/compiler.js";
+import markdownit from "markdown-it";
+
+export function getNamespaces(
+  documentation: Map<string, string> | undefined,
+): string[] {
+  if (documentation) {
+    return Array.from(documentation.keys());
+  }
+  return new Array<string>();
+}
+
+// Takes array of documents (containing data for each item in the standard library)
+// and creates a documentation map, which maps from a namespace
+// to the combined HTML-formatted documentation for all items in that namespace.
+export function processDocumentFiles(docFiles: DocFile[]): Map<string, string> {
+  const md = markdownit();
+  const contentByNamespace = new Map<string, string>();
+  const regex = new RegExp("^qsharp.namespace: Microsoft.Quantum.(.+)$", "m");
+
+  for (const doc of docFiles) {
+    const match = regex.exec(doc.metadata); // Parse namespace out of metadata
+    if (match == null) {
+      continue; // Skip items with non-parsable metadata
+    }
+    // The next line contains "Zero-width space" unicode character
+    // to allow line breaks before the period.
+    const newNamespace = "… " + match[1].replace(".", "​.");
```

Comment:
nit: I don't think the "... " is necessary and looks a bit strange to me given that the other sections of the nav don't use this syntax.

Reply:
Any suggestions? Keeping 'Microsoft.Quantum." in every line seems like a waste of space.

Reply:
I was thinking to just drop the "... ".

Reply:
That was the first thing I tried. It looked strange - seeing just Convert and similar names without anything in front didn't associate these names to namespaces for me...